BaaServiceKitTests/ProofTests/ProofModelTests.swift

```diff
@@ -0,0 +1,47 @@
+import XCTest
+@testable import BaaServiceKit
+
+class ProofModelTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    func testProofResponse() throws {
+        let file = Bundle(for: type(of: self)).url(forResource: "PartialFakeProofResponse", withExtension: "json")!
+        
+        do {
+            let data = try Data(contentsOf: file)
```

Comment:
if the test function `throws`, than you do not have to wrap the `try` into a `do - catch` statement